File: 1098.py
flag = 0

for A in range(3):
    if flag == 1: break
    for B in range(0, 9, 2):
        if flag == 1: break
        for C in range(A+1, A+4):
            if B != 0: print('I={}.{} J={}.{}'.format(A, B, C, B))
            else: print('I={} J={}'.format(A, C))
            if A == 2 and B == 0 and C == 5: flag = 1

File: 1098.cpp
#include <iostream>

using namespace std;

int main(void) {
    int A, B, C;
    int flag = 0;

    for (A = 0; A <= 2; ++A) {
        if (flag == 1) break;
        for (B = 0; B <= 8; B += 2) {
            if (flag == 1) break;
            for (C = A + 1; C <= A + 3; ++C) {
                if (B != 0) cout << "I=" << A << "." << B << " J=" << C << "." << B << endl;
                else cout << "I=" << A << " J=" << C << endl;
                if (A == 2 && B == 0 && C == 5) flag = 1;
            }
        }
    }
    return 0;
}
